ush/config.sh.RRFS_CONUS_13km.EnsFcst

```diff
@@ -0,0 +1,283 @@
+MACHINE="jet"
```

Comment:
Will the file config.sh.RRFS_CONUS_13km.EnsFcst be used when you add the EnsDA?  If so, to be more general, suggest to change the file name to "config.sh.RRFS_CONUS_13km.Ens"